README.md

```diff
@@ -5,10 +5,10 @@
 [![Publish Release to Maven Central](https://github.com/SDA-SE/sda-dropwizard-commons/workflows/Publish%20Release%20to%20Maven%20Central/badge.svg)](https://github.com/SDA-SE/sda-dropwizard-commons/actions?query=workflow%3A%22Publish+Release+to+Maven+Central%22)
 [![Publish Release to SDA](https://github.com/SDA-SE/sda-dropwizard-commons/workflows/Publish%20Release%20to%20SDA/badge.svg)](https://github.com/SDA-SE/sda-dropwizard-commons/actions?query=workflow%3A%22Publish+Release+to+SDA%22)
 [![FOSSA Status](https://app.fossa.com/api/projects/custom%2B8463%2Fsda-dropwizard-commons.svg?type=shield)](https://app.fossa.com/reports/2d8b4a40-db62-4c73-a978-588e252aa6e8)
-[![Quality Gate Status](https://sonarcloud.io/api/project_badges/measure?project=SDA-SE_sda-dropwizard-commons&metric=alert_status)](https://sonarcloud.io/dashboard?id=SDA-SE_sda-dropwizard-commons)
-[![SonarCloud Coverage](https://sonarcloud.io/api/project_badges/measure?project=SDA-SE_sda-dropwizard-commons&metric=coverage)](https://sonarcloud.io/dashboard?id=SDA-SE_sda-dropwizard-commons)
-[![SonarCloud Reliability](https://sonarcloud.io/api/project_badges/measure?project=SDA-SE_sda-dropwizard-commons&metric=reliability_rating)](https://sonarcloud.io/component_measures/metric/reliability_rating/list?id=SDA-SE_sda-dropwizard-commons)
-[![SonarCloud Security](https://sonarcloud.io/api/project_badges/measure?project=SDA-SE_sda-dropwizard-commons&metric=security_rating)](https://sonarcloud.io/component_measures/metric/security_rating/list?id=SDA-SE_sda-dropwizard-commons)
+[![Quality Gate Status](https://sonarcloud.io/api/project_badges/measure?project=SDA-SE_sda-dropwizard-commons&metric=alert_status)](https://sonarcloud.io/summary/new_code?id=SDA-SE_sda-dropwizard-commons)
+[![SonarCloud Coverage](https://sonarcloud.io/api/project_badges/measure?project=SDA-SE_sda-dropwizard-commons&metric=coverage)](https://sonarcloud.io/summary/new_code?id=SDA-SE_sda-dropwizard-commons)
+[![SonarCloud Reliability](https://sonarcloud.io/api/project_badges/measure?project=SDA-SE_sda-dropwizard-commons&metric=reliability_rating)](https://sonarcloud.io/summary/new_code?id=SDA-SE_sda-dropwizard-commons)
+[![SonarCloud Security](https://sonarcloud.io/api/project_badges/measure?project=SDA-SE_sda-dropwizard-commons&metric=security_rating)](https://sonarcloud.io/summary/new_code?id=SDA-SE_sda-dropwizard-commons)
```

Comment:
I see an error page when I follow the links [here](https://github.com/SDA-SE/sda-dropwizard-commons/blob/d6c84190ae9027c9189f01524a1bc03683affa37/README.md).

Reply:
I also see the following error at the moment:
```
403 ERROR
The request could not be satisfied.
Request blocked. We can't connect to the server for this app or website at this time. There might be too much traffic or a configuration error. Try again later, or contact the app or website owner.
If you provide content to customers through CloudFront, you can find steps to troubleshoot and help prevent this error by reviewing the CloudFront documentation. 
```

Let's wait for some minutes to see if the error just vanishes.

Reply:
I've double-checkd:
* the link is correct; you can copy the link and use it to open the page in a new browser tab, for example
* I guess the only difference is the "Referrer" header that contains the Github page; could be a rate limiting issue on Sonarcloud side?

Reply:
But is it a good idea to have a link that only works under certain circumstances?

Reply:
Can you modify the markdown link to not use the "Referrer"?